src/internal/execution.mligo

```diff
@@ -44,6 +44,8 @@ let send (content : proposal_content) (storage : storage_types)
     | Add_owners s -> ([], Storage.Op.add_owners s storage)
     | Remove_owners s -> ([], Storage.Op.remove_owners s storage)
     | Adjust_effective_period i -> ([], Storage.Op.adjust_effective_period i storage)
+    | Add_or_update_metadata { key; value } -> ([], Storage.Op.update_metadata (key, (Some value), storage))
```

Comment:
Wouldn't `Set_metadata` be shorter and a better naming? But maybe it's not as much explicit 🤔

Reply:
I was referring the batcher's contract for naming. I was bothering by it a little bit.